.eslintrc

```diff
@@ -14,9 +14,10 @@
   },
   "parser": "babel-eslint",
   "rules": {
-    "object-curly-spacing": 0,
-    "space-before-function-paren": [2, "never"],
-    "react/prefer-stateless-function": 0,
+    "arrow-body-style": "off",
```

Comment:
Yeah it's ok some of the time - other times I feel like it's annoying if the lack of braces and a return obscures the readability. I feel like this is one where being opionated gets in the way and I'd rather devs decide what makes sense on a case by case basis. 

The reason for disabling it was because it can't deal with the abiguity of returning an object since braces and the start of the object are both `{` - but I'll check and see if there's extra options to deal with this case before removing it entirely.